src/app.py

```diff
@@ -233,6 +257,22 @@ def update_histogram(value):
     Input('slider_saturation', 'value'),
     Input('slider_value_brightness', 'value'))
 def display_user_HSV_option(H, S, V):
+    """Display the user-selected HSV value as a str.
+
+    Parameters
+    ----------
+    H : int
+        Hue value selected by user via slider component.
+    S : int
+        Saturation value selected by user via slider component.
+    V : int
+        Value/Brightness value selected by user via slider component.
+
+    Returns
+    -------
+    str
```

Comment:
Actually, here will return a list of 2 items. The 1st one would be the HSV user choice prompt and the 2nd one would be a dictionary of CSS style for the rectangle to specify the color and shape.